src/polyfills.ts

```diff
@@ -61,4 +61,3 @@ import 'zone.js/dist/zone';  // Included with Angular CLI.
 /***************************************************************************************************
  * APPLICATION IMPORTS
  */
-import 'hammerjs/hammer';
```

Comment:
Isn't hammer js required by angular materials for better support for mobile devices?

Reply:
Since Angular 9 it is optional. It is required only when you use certain gestures on mobile. But in our case, we don't have any.